src/org/intellij/erlang/psi/impl/ErlangPsiImplUtil.java

```diff
@@ -836,6 +836,42 @@ public static ErlangFile resolveToFile(@Nullable ErlangModuleRef ref) {
     return ObjectUtils.tryCast(resolved != null ? resolved.getContainingFile() : null, ErlangFile.class);
   }
 
+  @NotNull
+  public static List<ErlangFunction> resolveToFunctions(@Nullable ErlangFunctionCallExpression call) {
+    PsiReference reference = call != null ? call.getReference() : null;
+    ResolveResult[] results = reference instanceof PsiPolyVariantReference ?
+                              ((PsiPolyVariantReference) reference).multiResolve(true) : ResolveResult.EMPTY_ARRAY;
+    return ContainerUtil.mapNotNull(results, new Function<ResolveResult, ErlangFunction>() {
+      @Override
+      @Nullable
+      public ErlangFunction fun(ResolveResult resolveResult) {
+        return ObjectUtils.tryCast(resolveResult.getElement(), ErlangFunction.class);
+      }
+    });
+  }
+
+  @NotNull
+  public static List<ErlangFunction> resolveToFunctions(@NotNull ErlangExpression argument) {
+    ErlangArgumentList argList = ObjectUtils.tryCast(argument.getParent(), ErlangArgumentList.class);
+    ErlangFunctionCallExpression call = ObjectUtils.tryCast(argList != null ? argList.getParent() : null,
+                                                            ErlangFunctionCallExpression.class);
+    return resolveToFunctions(call);
+  }
+
+  @Nullable
+  public static ErlangExpression getFunctionCallArgument(@Nullable PsiElement child) {
+    ErlangExpression argument = PsiTreeUtil.getParentOfType(child, ErlangExpression.class, false);
+    while (argument != null && !(argument.getParent() instanceof ErlangArgumentList)) {
+      argument = ObjectUtils.tryCast(argument.getParent(), ErlangExpression.class);
+    }
+    return argument;
```

Comment:
I've already mentioned that ErlangArgumentList is used not only in ErlangFunctionCall, but also in other elements. Please, implement this method properly.